### ISSUE TITLE
bug: @wagmi/cli 1.0.3  Did you mean to import abitype/dist/cjs/zod/index.js?

### ISSUE THREAD

**Issue by huazhuangnan** [wagmi-dev/wagmi#2450]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

1.0.3

### Current Behavior

run  `npx wagmi generate `
Cannot find module '...\node_modules\@wagmi\cli\node_modules\abitype\dist\esm\zod\index.mts' imported from ...\node_modules\@wagmi\cli\dist\commands-X2PX73GF.js
Did you mean to import abitype/dist/cjs/zod/index.js?

**Comment by tmm** [comment]
Need to bump ABIType to [`0.8.5`](https://github.com/wagmi-dev/abitype/releases/tag/abitype%400.8.5)

**Comment by lizonlukasz** [comment]
I've got exactly the same issue when trying to run `npx wagmi init`. Using `node v18.16.0` / `npm v9.5.1`

**Comment by tuler** [comment]
Is there a workaround? I tried to force `abitype` to `0.8.5` but it still doesn't work.

**Comment by MrKampla** [comment]
> Is there a workaround? I tried to force `abitype` to `0.8.5` but it still doesn't work.

While we wait for it to be fixed, I've created a little workaround ts script to edit the `package.json` exports of @wagmi/cli in `node_modules`:

```ts
#!/usr/bin/env ts-node --skipProject

import { readFile, writeFile } from 'node:fs/promises';
import { resolve } from 'node:path';

(async function () {
  console.log('Fixing abi-type package.json...');
  console.log(
    'This fix is needed because of a bug described in this issue https://github.com/wagmi-dev/wagmi/issues/2288',
  );
  const abiTypePackageJsonPath = resolve(
    __dirname,
    '../../../node_modules/abitype/package.json',
  );
  const packageJson = JSON.parse(await readFile(abiTypePackageJsonPath, 'utf-8'));

  const { exports } = packageJson;

  const zodExports = exports['./zod'];

  exports['./zod'] = {
    ...zodExports,
    import: './dist/esm/zod/index.js',
  };

  await writeFile(abiTypePackageJsonPath, JSON.stringify(packageJson, null, 2));
})();
```

Just make sure to pass a correct path to the `package.json` (I'm using a monorepo, hence so many parent directory references). Run this script before using wagmi cli commands.

**Comment by jxom** [comment]
PR to fix: #2484

**Comment by jxom** [comment]
I have published a prerelease version at `wagmi@0.0.0-20230602023817` if you'd like the fix now!

**Comment by jxom** [comment]
Released on `1.1.0`!
